openapi.yml

```diff
@@ -281,6 +281,12 @@ components:
           type: array
           items:
             type: string
+    MaintenanceWindowCreate:
```

Comment:
[Swagger.io editor](https://editor.swagger.io/) is pointing out errors on how these attrs were added, could you fix it, @Ktmi? If you could also add a description to each property explaining what they're for that'd be great. 

![20230626_155127](https://github.com/kytos-ng/maintenance/assets/1010796/05771423-5e96-4473-9529-d3f16e796904)

Reply:
Should be fixed. As nice as it would be to just say the type, it instead asks for an object which has the type as an attr :(